code/game/machinery/doors/door.dm

```diff
@@ -226,7 +249,28 @@
 /obj/machinery/door/proc/try_to_crowbar(mob/user, obj/item/I)
 	return
 
+/obj/machinery/door/proc/clean_cmag_ooze(obj/item/I, mob/user) //Emags are Engineering's problem, cmags are the janitor's problem
```

Comment:
So this is.. alright for the moment, but I don't want this turning into attackby 2 with a big amount of ifs for cleaning items. In the future it should be handled on the attacking item (soap, etc.), and use `clean_act` or something like we were all talking about on discord the other day.

Reply:
Will be handling this in a future PR that standardizes cleaning stuff, thanks!